cmd/agent/app/testdata/test_config.yaml

```diff
@@ -20,7 +20,7 @@ processors:
         maxPacketSize: 65001
         hostPort: 3.3.3.3:6832
 
-samplingServer:
+httpServer:
```

Comment:
so this will break out internal configs in Puppet? Maybe we should prep a change ahead of time.